### PR TITLE
Added examples 1.25, 1.26.  Added \todoex{}

### PR DESCRIPTION
**Pull request by jensenr30**
I added \todoex{} command to show where solutions are missing
Changed 1/4 to 0.25 because the compiler was throwing a fit saying it
was not a quantity.

### CODE REVIEW

**Comment by jensenr30** [comment]
@milesdai 
If you would like me to not bug you so much, I can submit pull requests that contain solutions to more exercises at a lower frequency.  Let me know what you think.

File: Chapter1.tex [hunk 1]
```diff
@@ -509,8 +507,66 @@ \chapter{Solutions for Chapter 1}
...
+    The impedance of 2 parallel capacitors is equal to the impedance of a single capacitor $C$ of value $C_1 + C_2$:
+    \[ \mathbf{Z_\text{parallel}} = \frac{1}{ \frac{1}{\mathbf{Z_1}} + \frac{1}{\mathbf{Z_2}} } = \frac{1}{ j\omega C_1 + j\omega C_2 } = \mans{\frac{1}{j\omega (C_1 + C_2)}} \]
+    
+    \item
+    The impedance of 2 series capacitors is equal to the impedance of a single capacitor $C$ of value $\frac{C_1 C_2}{C_1 + C_2}$:
+    \begin{equation*}
+        \begin{aligned}
+            \mathbf{Z_{\text{series}}} = \mathbf{Z_1} + \mathbf{Z_2} =  \frac{1}{j\omega C_1} + \frac{1}{j\omega C_2} = \frac{1}{j\omega} \left( \frac{1}{C_1} + \frac{1}{C_2} \right) = \frac{1}{j\omega} \left( \frac{C_2}{C_1 C_2} + \frac{C_1}{C_2 C_1} \right) \\= \frac{1}{j\omega} \left( \frac{C_1 + C_2}{C_1 C_2} \right) = \frac{1}{j\omega} \frac{1}{\left( \frac{C_1 C_2}{C_1 + C_2} \right)} = \mans{ \frac{1}{j\omega\left( \frac{C_1 C_2}{C_1 + C_2} \right)}}
+        \end{aligned}
+    \end{equation*}
```
Comment (milesdai) [inline Chapter1.tex:524]: I suggest using the starred `align` environment here provided by the amsmath package and splitting this line into 1 equal sign per line. See problem 1.14 for an example.

```suggestion
    \begin{align*}
        \mathbf{Z_{\text{series}}} &= \mathbf{Z_1} + \mathbf{Z_2} \\
        &=  \frac{1}{j\omega C_1} + \frac{1}{j\omega C_2} \\
        &= \frac{1}{j\omega} \left( \frac{1}{C_1} + \frac{1}{C_2} \right) \\
        &= \frac{1}{j\omega} \left( \frac{C_2}{C_1 C_2} + \frac{C_1}{C_2 C_1} \right) \\
        &= \frac{1}{j\omega} \left( \frac{C_1 + C_2}{C_1 C_2} \right) \\
        &= \frac{1}{j\omega} \frac{1}{\left( \frac{C_1 C_2}{C_1 + C_2} \right)} \\
        &= \mans{ \frac{1}{j\omega\left( \frac{C_1 C_2}{C_1 + C_2} \right)}}
    \end{align*}
```

**Review by milesdai — CHANGES_REQUESTED** [review]
Just one comment about aligned equations. LGTM otherwise!

**Comment by milesdai** [comment]
> @milesdai If you would like me to not bug you so much, I can submit pull requests that contain solutions to more exercises at a lower frequency. Let me know what you think.

Don't worry about spamming me -- I'd rather keep the PRs modular so that it's easier to deal with each one individually. Appreciate the contributions!

File: Chapter1.tex (hunk 1 above)
Comment (jensenr30) [reply Chapter1.tex:524]: That look much nicer in code and on paper.

**Pull request merged**